delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/routes/ElasticSearchViewsRoutes.scala

```diff
@@ -275,36 +274,30 @@ final class ElasticSearchViewsRoutes(
 
   private def genericResourcesRoutes(implicit caller: Caller): Route =
     pathPrefix("resources") {
-      projectRef(projects).apply {
-        case ProjectRef(_, project) if project.value == "events" => reject()
```

Comment:
we don't need this guard anymore?

Reply:
No. Look at the PR on `UriDirectives.label` and `UriDirectives.idSegment`